bench/tests/test_init.py

```diff
@@ -186,7 +186,7 @@ def drop_site(self, site_name, archived_sites_path=None):
 		if archived_sites_path:
 			drop_site_cmd.extend(['--archived-sites-path', archived_sites_path])
 
-		if os.environ.get('TRAVIS'):
+		if os.environ.get('CI'):
```

Comment:
`if os.environ.get("TRAVIS") or os.environ.get('CI'):` would be better I guess, in case we do go back.